admin/bin/qserv-data-loader.py

```diff
@@ -52,9 +52,9 @@
 
 """
 
-#--------------------------------
+# -------------------------------
```

Comment:
I think your editor is over-compliant. The section that you quote is about comments (like in documenting), the line:

    #--------------------------------

is not a comment. Again, even if your editor is 100% compliant with some guidelines it does not mean it's a good reason to introduce changes that do not actually improve things.

It's OK to not revert this file (because it's merged with master already) but please avoid doing this in the future.